cpu/atmega1284p/periph/rtt.c

```diff
@@ -0,0 +1,131 @@
+#include "board.h"
+#include "periph/rtt.h"
+#ifdef MODULE_PM_LAYERED
+#include "pm_layered.h"
+#endif
+
+#include "irq.h"
+#include "thread.h"
+
+rtt_cb_t rtt_next_cb;
+volatile int cnt = 0;
+
+void rtt_init(void) {
```

Comment:
You could split out `periph/rtt` as its own PR. Without having looked into it closely, I guess this should also be possible to add to `cpu/atmega_common` instead.